src/index.js

```diff
@@ -245,22 +245,28 @@ export default class EmojiReact extends Component {
 	}
 
 	render() {
-		const { reactions, onReaction, onEmojiClick } = this.props;
+		let { reactions, onReaction, onEmojiClick, sort } = this.props;
+		if (sort) {
+		reactions = reactions.slice(0);
+	    reactions.sort(function (a, b) {
+	      return b.count - a.count;
```

Comment:
Fixed. Yes I think it makes more sense to (be able to) sort emojis by most used.